src/script/utils/breakpoints.ts

```diff
@@ -0,0 +1,49 @@
+import { unsafeCSS, CSSResult } from 'lit-element';
```

Comment:
Breakpoint helper!
The files at this point are mostly css, so I thought it'd be nice to at least cut out some boiler plate.
We can use the same concept to modularize parts of our css too! Not 100% sold on this implementation, but I stored in the git history an iteration I tried, it worked but it proved to be less intuitive.

https://github.com/pwa-builder/PWABuilder/pull/1351/commits/4a1168fbfda90826c67668e1da03b0226a8b481a